### PR TITLE
PSM: Don't read padding parameters from private namespace

### LINKED ISSUE #2699: PSM ros params wrongly needs namespace prefix
Quoting @mintar from [here](https://github.com/ros-planning/moveit/issues/1330#issuecomment-851349546):

> Just a further note for everyone else who stumbles upon this: I'm using a prefix (mobipick) for my robot. It used to work exactly how they described:
> 
> mobipick/move_group/robot_description_planning/default_attached_padding: 0.0
> 
> In the current ROS Noetic version of MoveIt (1.1.5), there's a second mobipick necessary to make it work:
> 
> mobipick/move_group/mobipick/robot_description_planning/default_attached_padding: 0.0

Looking into it a bit this is probably a bug introduced in Henning's efforts to have `move_group` use `MoveItCpp` for support of multiple planning pipelines.
Specifically, [this commit](https://github.com/ros-planning/moveit/pull/2127/commits/8df40df56ad58a6358834129b09b3a8b2c35a02c) replaces `move_group`'s own PlanningSceneMonitor with the one initialized by MoveItCpp.

Please have a look @henningkayser .

### PR DESCRIPTION
**Pull request by mintar**
Before this commit, the padding parameters were read from the private namespace of the move_group node (unlike all the other PSM parameters).

Fixes #2699.

### CODE REVIEW

**Review by rhaschke — COMMENTED** [review]
`robot_description` is the same as `robot_description_` except having a leading slash removed.
I guess that in most cases, they are identical. Or do we store an _absolute_ param path in `robot_description_`?

**Comment by mintar** [comment]
> do we store an absolute param path in robot_description_?

Yes. See #1330 .

**Comment by mintar** [comment]
I just remembered that I need to fix a small thing tomorrow, don't merge yet.

**Comment by mintar** [comment]
Ok, the thing I just changed is to make sure that the depreciation warning is only emitted when the `/robot_description` parameter is an absolute path. Otherwise, `old_robot_description` and `robot_description_` are identical, so it makes no sense printing that warning.

Back to your question: 

>  Or do we store an absolute param path in robot_description_?

This is actually more convoluted than I thought. Here goes:

On my robot, I'm starting `move_group` in the namespace `/mobipick`. I'm going to use this in the rest of my comment because it's the more general case, but it works the same without namespaces if you replace `/mobipick` with the empty string in the rest of my comment.

The PSM gets `robot_description_` from the `RobotModelLoader`, here:

https://github.com/ros-planning/moveit/blob/dba2095a0b4a4a2352c8991a779bafe11abeefc9/moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp#L189

... which in turn gets it from the `RDFLoader`, here:

https://github.com/ros-planning/moveit/blob/dba2095a0b4a4a2352c8991a779bafe11abeefc9/moveit_ros/planning/robot_model_loader/include/moveit/robot_model_loader/robot_model_loader.h#L95

... which uses `searchParam("robot_description", ...)` to get the parameter:

https://github.com/ros-planning/moveit/blob/3361b2d1b6b2feabc2d3e93c75653f5a00e87fa4/moveit_ros/planning/rdf_loader/src/rdf_loader.cpp#L67

The parameter layout on the parameter server looks like this:

```
/mobipick/robot_description: <URDF>
/mobipick/move_group/...: some more parameters
```

Since `searchParam` cannot find `/mobipick/move_group/robot_description`, it searches upward and finds ` /mobipick/robot_description` (fully resolved, i.e., with a leading `"/"`).

So from now on, `robot_description_ == " /mobipick/robot_description"` in the PSM.

This is now used by the PSM to load some parameters:

https://github.com/ros-planning/moveit/blob/dba2095a0b4a4a2352c8991a779bafe11abeefc9/moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp#L247

... and some more, except for the padding parameters, where it drops the leading `"/"`:

https://github.com/ros-planning/moveit/blob/dba2095a0b4a4a2352c8991a779bafe11abeefc9/moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp#L1426-L1429

This results in the following parameters being read:

```
/mobipick/robot_description_planning/cartesian_limits
/mobipick/robot_description_planning/default_collision_operations
/mobipick/robot_description_planning/joint_limits
/mobipick/robot_description_planning/shape_transform_cache_lookup_wait_time
/mobipick/move_group/mobipick/robot_description_planning/default_attached_padding
/mobipick/move_group/mobipick/robot_description_planning/...
```

This PR changes this so the following parameters are read instead:

```
/mobipick/robot_description_planning/cartesian_limits
/mobipick/robot_description_planning/default_collision_operations
/mobipick/robot_description_planning/joint_limits
/mobipick/robot_description_planning/shape_transform_cache_lookup_wait_time
/mobipick/robot_description_planning/default_attached_padding
/mobipick/robot_description_planning/...
```

**Comment by v4hn** [comment]
As Robert noticed after I merged #2703 you targeted `noetic-devel` but we merge things into `master` and `noetic-devel` is still rebased every now and then.
I retargeted your two related requests.

**Review by v4hn — APPROVED** [review]

**Pull request merged**
